blocks/library/list/index.js

```diff
@@ -59,6 +59,19 @@ registerBlockType( 'core/list', {
 					} );
 				},
 			},
+			...[ 'OL', 'UL' ].map( ( tag ) => ( {
+				type: 'shortcut',
+				blocks: [ 'core/paragraph' ],
+				shortcut: tag.charAt( 0 ).toLowerCase(),
```

Comment:
Unless we centralize shortcut declarations in one place, perhaps add a line comment spelling out which shortcuts this adds, to assist the reader and to improve codebase searchability: `// Keyboard shortcut: access+o, access+u`

Reply:
I will do the same as the alignment here, for readability.